docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,286 @@
+started?](#what-should-i-know-before-i-get-started)
+
+[How Can I Contribute?](#how-can-i-contribute)
+  * [Reporting Bugs](#reporting-bugs)
+  * [Suggesting Enhancements](#suggesting-enhancements)
+  * [Your First Code Contribution](#your-first-code-contribution)
+  * [Pull Requests](#pull-requests)
+
+## Code of Conduct
+
+This project and everyone participating in it is governed by the \ref
+code_of_conduct "SpECTRE Code of Conduct". By participating, you are
+expected to uphold this code. Please report unacceptable behavior to
+[conduct@spectre-code.org](mailto:conduct@spectre-code.org).
+
+## What should I know before I get started?
+
+SpECTRE is being developed in support of our collaborative Simulating
+eXtreme Spacetimes (SXS) research program into the multi-messenger
+astrophysics of neutron star mergers, core-collapse supernovae, and
+gamma-ray bursts.  As such, almost all of the contributors to SpECTRE
+are members of SXS institutions, and a large amount of planning the
+design of SpECTRE is done in internal SXS meetings.  If you are a
+member of SXS and wish to get involved, please contact one of the
+project leaders.
+
+In the future, we hope that SpECTRE can be applied to problems across
+discipline boundaries, and that it can be a true community code.  At
+the present time, however, SpECTRE lacks the capability to solve
+realistic problems, and broad overview documentation is incomplete.
+Therefore, if you are not a member of SXS, but are interested in
+contributing to SpECTRE, we strongly encourage you to contact us at
+questions@spectre-code.org to discuss possible contributions.
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+This section guides you through submitting a bug report for
+SpECTRE. Following these guidelines helps maintainers and the
+community understand your report, reproduce the behavior, and find
+related reports.
+
+Before creating bug reports, please **perform a
+[search](https://github.com/sxs-collaboration/spectre/issues)** to see
+if the problem has already been reported. If it has **and the issue is
+still open**, add a comment to the existing issue instead of opening a
+new one.
+
+> **Note:** If you find a **Closed** issue that seems like it is the
+> same thing that you're experiencing, open a new issue and include a
+> link to the original issue in the body of your new one.
+
+#### How Do I Submit A (Good) Bug Report?
+
+Bugs are tracked as [GitHub
+issues](https://guides.github.com/features/issues/). When you are
+creating a bug report, please **include as many details as
+possible**. Fill out the required template, as the information it asks
+for helps us resolve issues faster.
+
+Explain the problem and include additional details to help maintainers
+reproduce the problem:
+
+* **Use a clear and descriptive title** for the issue to identify the
+  problem.
+* **Describe the exact steps which reproduce the problem** in as many
+  details as possible. For example, start by explaining how you
```

Comment:
"in as many details as possible" can be rephrased to "in as much detail as possible"